src/main/java/org/kohsuke/github/GHPullRequest.java

```diff
@@ -110,7 +104,7 @@ public GHCommitPointer getBase() {
     public GHCommitPointer getHead() {
         return head;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this line start at column 5. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AIndentationCheck)